docs/src/main/asciidoc/core.adoc

```diff
@@ -24,6 +24,15 @@ dependencies {
 }
 ----
 
+=== Configuration
+
+The following options may be configured with Spring Cloud core.
+
+|===========================================================================
+| Name | Description | Required | Default value
+| `spring.cloud.gcp.core.enabled` | Enables or disables GCP core auto configuration | No | `true`
```

Comment:
Since we're adding this table, (which we should've always had), we should probably also list `spring.cloud.gcp.credentials.location`, `spring.cloud.gcp.credentials.encoded-key`, `spring.cloud.gcp.project-id`, and `spring.cloud.gcp.credentials.scopes`.

I understand that's not the main focus of this PR. So, I would be OK with it being done in a separate PR.

Reply:
I filed a follow-up issue #2151 for this.

Reply:
Better to do it in a separate PR connected to the new issue #2151 , I guess.